Xamarin.PropertyEditing.Mac/PropertyTableDelegate.cs

```diff
@@ -17,6 +17,11 @@ internal class PropertyTableDelegate
 		public PropertyTableDelegate (PropertyTableDataSource datasource)
 		{
 			this.dataSource = datasource;
+
+			// Add the Mac specific mappings
```

Comment:
These shouldn't be added back in.

Reply:
Agreed. I added them back in so I could test the AppKit, but if we aren't supporting that anymore.